lametro/templates/partials/search_bar.html

```diff
@@ -20,3 +22,34 @@
         <input type="radio" name="search-reports" autocomplete="off" {% if 'search-reports' in request.GET %} checked{% endif %}> Search reports only
     </label>
 </div>
+
+{% block extra_js %}
```

Comment:
putting this block here doesn't have quite the effect we want: it puts the javascript in the middle of the page. 

in order to add this to the end of the full search markup, you need to extend [the default `search.html` template](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/templates/search/search.html). this will involve creating a `search.html` file at the same path in the `lametro/templates`, and putting these script elements inside an extended `extra_js` block. 

there's more guidance in [the django docs on template inheritance](https://docs.djangoproject.com/en/2.0/ref/templates/language/#template-inheritance). one thing in particular you will find useful in adding content to a block is this:

> If you need to get the content of the block from the parent template, the `{{ block.super }}` variable will do the trick. This is useful if you want to add to the contents of a parent block instead of completely overriding it.

(i just learned this looking for the docs to add to this comment! 🙃)

Reply:
thank you! it didn't occur to me that the base search template would be a part of councilmatic. lacking that it seemed this was the best place over `base.html`.